cdk/lib/recipes-backend.ts

```diff
@@ -147,13 +153,55 @@ export class RecipesBackend extends GuStack {
         statistic: "Maximum",
         unit: Unit.MILLISECONDS
       }),
-      evaluationPeriods: 3,// when happens atleast 3 times
+      evaluationPeriods: 3,// when happens at least 3 times
     })
 
-    const externalParameters = new ExternalParameters(this, "externals");
-    //const urgentAlarmTopic = aws_sns.Topic.fromTopicArn(this, "urgent-alarm", externalParameters.urgentAlarmTopicArn.stringValue);
-    const nonUrgentAlarmTopic = aws_sns.Topic.fromTopicArn(this, "nonurgent-alarm", externalParameters.nonUrgentAlarmTopicArn.stringValue);
-    durationAlarm.addAlarmAction(new SnsAction(nonUrgentAlarmTopic))
+    durationAlarm.addAlarmAction(new SnsAction(nonUrgentAlarmTopic));
+
+    const publishTodaysCurationLambda = new GuScheduledLambda(this, "PublishTodaysCuration", {
+      app: "recipes-publish-todays-curation",
+      architecture: Architecture.ARM_64,
+      fileName: "publish-todays-curation.zip",
+      functionName: `PublishTodaysCuration-${props.stage}`,
+      handler: "main.handler",
+      initialPolicy: [
+        new PolicyStatement({
+          effect: Effect.DENY,
+          actions: ["*"],
+          resources: [serving.staticBucket.bucketArn + "/content/*"]
+        }),
+        new PolicyStatement({
+          effect: Effect.ALLOW,
+          actions: ["s3:PutObject","s3:GetObject"],
+          resources: [serving.staticBucket.bucketArn + "/*"]
+        }),
+        new PolicyStatement({
+          effect: Effect.ALLOW,
+          actions: ["s3:ListBucket"],
+          resources: [serving.staticBucket.bucketArn]
+        })
+      ],
+      memorySize: 256,
+      monitoringConfiguration: {
+        noMonitoring: true,
+      },
+      rules: [{
+        schedule: Schedule.cron({hour: "0", minute: "1"}),
+        description: "Update Feast app daily curation at midnight"
```

Comment:
A quick note that we may need different schedules for different editions. I wonder if, in future, we might delegate to clients, which will have their own opinion about when midnight is. This'll be fine for now.

Reply:
Yeah I'd like it with clients..... but as discussed this is OK for now.